wire-ios/Wire-iOS Tests/ConversationList/ConversationListCell/ConversationStatusLineTests.swift

```diff
@@ -123,7 +123,9 @@ class ConversationStatusLineTests: CoreDataSnapshotTestCase {
         // GIVEN
         let sut = self.otherUserConversation!
         for index in 1...5 {
-            (try! sut.appendText(content: "test \(index)") as! ZMMessage).sender = self.otherUser
+            let message = try! sut.appendText(content: "test \(index)") as! ZMClientMessage
```

Comment:
let's make the method throwable and replace `try!` by `try`

Reply:
@netbe Done!